src/Icons/Plush.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+
+interface Props {
+  width?: string | number
+  height?: string | number
+  fillColor?: string
+  [x: string]: unknown // for the rest property
+}
+
+const Plush: React.FunctionComponent<Props> = ({ width = '24', height = '15', fillColor = '#2D2926', ...rest }) => (
+  <svg width="32" height="33" viewBox="0 0 32 33" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path
+      fill-rule="evenodd"
+      clip-rule="evenodd"
```

Comment:
clippy and filly

Reply:
what do you mean by this?

Reply:
lol. fill-rule --> fillRule. clip-rule --> clipRule